cloud/filestore/libs/vfs_fuse/fs_impl.cpp

```diff
@@ -146,8 +146,15 @@ bool TFileSystem::UpdateNodeCache(
 
         entry.ino = attrs.GetId();
         entry.generation = NodeCache.Generation();
+
+        const auto nodeType = attrs.GetType();
+        const auto entryTimeout = (
+            nodeType == NProto::ENodeType::E_REGULAR_NODE ?
+            Config->GetRegularFileEntryTimeout().SecondsFloat() :
```

Comment:
i think it is safer to set it to zero by default and allow override only if it is set to non-zero value